app/models/db/enrollment_projection_by_boro.rb

```diff
@@ -0,0 +1,3 @@
+class Db::EnrollmentProjectionByBoro < DataRecord
+  self.table_name = DataPackage.latest_for(:public_schools).table_name_for(:enrollment_projection_by_boro)
+end
```

Comment:
```suggestion
end

```

(We should add a linter lol)